test_main.py

```
@@ -54,21 +56,88 @@
-[Link](http://example.com)
+        # Basic validation
+        assert "Format Test" in json_content
+        assert "Test Heading" in json_content
+        assert "Test paragraph" in json_content
+        assert "Item A" in json_content
+        assert "Item B" in json_content
 
-![Test Image](image.jpg)
+        # XML output test
+        xml_content = convert_html(
+            mock_response.text, "http://example.com", OutputFormat.XML
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)